www/firebase-crash.js

```diff
@@ -0,0 +1,13 @@
+var exec = require('cordova/exec');
+
+exports.logError = function (message, success, error) {
+  exec(success, error, "FirebaseCrashPlugin", "logError", [message]);
+};
+
+exports.sendNonFatalCrash = function (message, success, error) {
+  exec(success, error, "FirebaseCrashPlugin", "sendNonFatalCrash", [message]);
+};
+
+exports.setCrashlyticsUserId = function (userId, success, error) {
```

Comment:
setUserId

Reply:
I was going for naming the javascript method names equal to the java/objective c method names. If you prefer a short method name, I can change that. What do you think?

Reply:
Keep in mind that plugin api has namespace:
```js
cordova.plugins.firebase.crashlytics.setUserId(userId);
```
Having crashlytics word in a method name is redundant.

Reply:
Okay, I will correct that.